src/ripple/app/main/Main.cpp

```diff
@@ -358,50 +606,62 @@ main(int argc, char* argv[])
 
     if (lgrInfo.txHash != transactionHash)
         throw std::runtime_error(
-            "COULD NOT VERIFY LEDGER TX " + std::to_string(startingSequence));
+            "COULD NOT VERIFY INITAL LEDGER TX " + std::to_string(startingSequence));
 
     if (lgrInfo.accountHash != stateHash)
         throw std::runtime_error(
-            "COULD NOT VERIFY LEDGER ACCOUNT HASH " +
+            "COULD NOT VERIFY INITAL ACCOUNT HASH " +
             std::to_string(startingSequence));
 
     std::cout << "VERIFIED LEDGER: " << std::to_string(startingSequence)
               << std::endl;
 
-    while (startingSequence < endSequence)
-    {
-        ++startingSequence;
-        auto [header, txs, diffs] =
-            fetchTransactionsAndDiff(url, port, startingSequence);
-
-        transactionHash = hashTransactions(txs);
-
-        if (header.txHash != transactionHash)
-            throw std::runtime_error(
-                "COULD NOT VERIFY LEDGER TX " +
-                std::to_string(startingSequence));
 
-        for (auto const& diff : diffs)
+    std::thread dataVerifier([&]()
+    {
+        while (startingSequence < endSequence)
```

Comment:
can we make this run forever? Meaning don't pass in an end sequence, and just have the script verify indefinitely, as ledgers are validated?